Makefile

```diff
@@ -416,6 +418,17 @@ _debug-python-release: _build-release-image-python
 
 ################################################################################################################################################################
 
+
+clang-format-source: start-development-debian ## Format all of the source code with the rules in .clang-format
+	@ echo Running Clang Format. See .clang-format for Rules
+	$(shell clang-format -i -style=file:thirdparty/clang/.clang-format ${clang_format_sources_path})
+
+clang-format-check: start-development-debian ## Runs the clang-format tool to check the code against rules and formatting
```

Comment:
I assume we'll add a CI job (+ apply format to all files) once we have a consensus on the configuration, right?

Reply:
Yeah for sure.